Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -159,7 +127,7 @@ function Validate-AzurePowershellVersion
 
     if(!$versionCompatible)
     {
-        Write-TaskSpecificTelemetry "PREREQ_UnsupportedAzurePSVersion"
+        Write-Telemetry "Task_InternalError" "UnsupportedAzurePSVersion"
```

Comment:
I assume internal errors are also bad and it is actionable by us. Correct? If so, do we have enough info logged to take corrective action.